tasks/copy_to_archive/copy_to_archive.py

```diff
@@ -8,10 +8,12 @@
 from datetime import datetime, timezone
 from typing import Any, Dict, List, Union
 
-# Third party libraries
 import boto3
```

Comment:
Is boto3 not a third party library?

Reply:
It is. Added back under # Third party libraries